templates/web/base/govuk/fields.html

```diff
@@ -195,6 +195,7 @@ <h2 class="govuk-fieldset__heading">
       [% END %]
       [% IF item.tags.divider %]
         <p class="govuk-body"><strong>[% item.tags.divider %]</strong></p>
+        [% INCLUDE 'govuk/divider_intro.html' %]
```

Comment:
I have a feeling this will fail if/when this code runs on other cobrands which don't have this template.

Probably worth wrapping it in a try/catch, like we do in other places:

```
[% TRY %][% INCLUDE 'govuk/divider_intro.html' %][% CATCH file %][% END %]
```

Reply:
Have changed `divider` to `divider_template` to use as a template include (given added text anyway), then this won't matter as will only be used when that template exists, and made it easier to customise.